logstash-core/lib/logstash/util/wrapped_synchronous_queue.rb

```diff
@@ -234,9 +237,10 @@ def each(&blk)
 
         # below the checks for @cancelled.include?(e) have been replaced by e.cancelled?
         # TODO: for https://github.com/elastic/logstash/issues/6055 = will have to properly refactor
-        @originals.each do |e, _|
+        for e in @originals
```

Comment:
@jordansissel because `@originals` is a Java `HashSet` now and not a `Map`/`Hash` anymore